index.js

```diff
@@ -123,12 +116,30 @@ module.exports = function standard (opts) {
   }
 
   function lint () {
-    parallel([
-      spawn.bind(undefined, JSCS, jscsArgs),
-      spawn.bind(undefined, ESLINT, eslintArgs)
-    ], function (err, r) {
+    findBinPaths(function (err, paths) {
       if (err) return error(err)
-      if (r.some(Boolean)) printErrors()
+      parallel([
+        spawn.bind(undefined, paths.jscs, jscsArgs),
+        spawn.bind(undefined, paths.eslint, eslintArgs)
+      ], function (err, r) {
+        if (err) return error(err)
+        if (r.some(Boolean)) printErrors()
+      })
+    })
+  }
+
+  function findBinPaths (cb) {
+    parallel({
+      eslint: findBinPath.bind(undefined, 'eslint'),
+      jscs: findBinPath.bind(undefined, 'jscs')
+    }, cb)
+  }
+
+  function findBinPath (bin, cb) {
+    var opts = { cwd: __dirname }
+    cp.exec('npm run --silent which-' + bin, opts, function (err, stdout, stderr) {
+      if (err) return cb(err)
+      cb(null, stdout.toString().replace(/\n/g, ''))
```

Comment:
I prefer `String(stdout)` since that guarantees the return type, whereas `toString` does not - but since it's from `child_process.exec` it doesn't matter that much.